### PR TITLE
[CARBONDATA-703] Update build command after optimizing thrift compile issues

### PR DESCRIPTION
**Pull request by chenliang613**
Update build command after optimizing thrift compile issues

### CODE REVIEW

File: build/README.md [hunk 1]
```diff
@@ -57,3 +48,9 @@ Build with test
 ```
 mvn clean package
 ```
+
+## For any contributors : if change format code, need build as per the command
```
Comment (ravipesala) [inline build/README.md]: Better phrase as below.
For contributors : To build the format code after any changes, please follow the below command.

Comment (chenliang613) [inline build/README.md]: ok

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.6.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/896/

**Comment by ravipesala** [comment]
LGTM

**Pull request closed without merging**